README.md

```diff
@@ -30,6 +30,7 @@ $ npm run dev
 - Create a new file: `config/local.js`. Override the required config values from `config/development.js` and `config/default.js` into `config/local.js`.
 - Register the application for [GitHub OAuth](https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps) to get the `clientId` and `clientSecret`. Add the callback URL as `http://<HOSTNAME>:<PORT>/auth/github/callback`
 - Create an application on [FireStore](https://firebase.google.com/docs/firestore) and [generate a service file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys). Add the service file with the name `firestore-private-key.json` in the project root. 
+- Make sure **your** local development setup is working properly by running the following command: `npm run test-config`.
```

Comment:
`your` does not need to be in bold. I had made it bold just so that you can easily spot it

Also, what about adding this command here? https://github.com/Real-Dev-Squad/website-backend/blob/develop/CONTRIBUTING.md#npm-command-reference

Reply:
Done. Please merge!

Reply:
Also, what about adding this command here? https://github.com/Real-Dev-Squad/website-backend/blob/develop/CONTRIBUTING.md#npm-command-reference

Reply:
@swarajpure https://github.com/Real-Dev-Squad/website-backend/pull/112